docs/en/stack/ml/df-analytics/dfanalytics-limitations.asciidoc

```diff
@@ -133,8 +132,8 @@ stuck at 99% for several hours.
 
 If a reduction in runtime is important to you, try strategies such as disabling 
 feature importance, using a smaller {transform}, setting 
```

Comment:
rather than "using a smaller {transform}" I'd say something like "reducing the number of training examples (for example, by decreasing the training percentage".